Sources_v3/Utils/Internet Connection/InternetConnection.swift

```diff
@@ -45,6 +49,10 @@ class InternetConnection {
         self.monitor.delegate = self
     }
     
+    deinit {
+        stop()
+    }
+    
     func start() {
         monitor.start()
     }
```

Comment:
Why we don't start listening automatically when the object is created? Seems to me like an unnecessary complication. `start()` and `stop()` imo should even be public methods 🤷‍♂️

Reply:
IMHO
1. `WebSocketClient` should handle Internet connection when it's ready, user and token should be setupped.
2. On Client disconnect (basically logout) we shouldn't do anything related to WebSocket connect or anything else.
3. It's internal work/responsibility to handle connection until user do logout.
4. I'm not sure about public API, probably it's good to share it for UI, but `Client.internetConnection` should be internal var.